src/tests/api-tests.ts

```diff
@@ -4928,6 +4948,7 @@ describe('api tests', () => {
       burn_block_height: 123,
       miner_txid: '0x4321',
       canonical: true,
+      total_execution_cost: 0,
```

Comment:
Could you add (or modify) some tests to make the total execution cost non-zero? In order to exercise the reduce function.